image/l0_smoothing/l0_smoothing.py

```diff
@@ -0,0 +1,105 @@
+from typing import Callable
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver/module does.